### PR TITLE
Add PB profile shape for discussion

### PR DESCRIPTION
**Pull request by VirginiaBalseiro**
This PR adds the ShEx shape used in PodBrowser for discussion. 

Note: I think we want to go with SHACL but this is just to get us started with discussion.

### CODE REVIEW

**Comment by csarven** [comment]
Related to https://github.com/solid/webid-profile/issues/26

File: shapes/profile.shex [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    vcard:Pref
+    vcard:Work
+    vcard:X400
+  ] ?
+    // rdfs:comment  "They type of Phone Number" ;
+  vcard:value IRI
+    // rdfs:comment  "The value of a phone number as a tel link (Example <tel:555-555-5555>)" ;
+}
+
+srs:TrustedAppShape {
```
Comment (elf-pavlik) [inline shapes/profile.shex:119]: I think this shouldn't be included until there is a draft for trusted apps

File: shapes/profile.shex [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+PREFIX schem: <http://schema.org/>
+PREFIX vcard: <http://www.w3.org/2006/vcard/ns#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX acl: <http://www.w3.org/ns/auth/acl#>
+PREFIX cert:  <http://www.w3.org/ns/auth/cert#>
+PREFIX ldp: <http://www.w3.org/ns/ldp#>
+PREFIX sp: <http://www.w3.org/ns/pim/space#>
+PREFIX solid: <http://www.w3.org/ns/solid/terms#>
+
+srs:SolidProfileShape EXTRA a {
```
Comment (woutermont) [inline shapes/profile.shex:13]: This shape describes a Person, so I suggest we call it the `srs:PersonShape`. Optionally, I would then add a `srs:SolidProfileShape`, that describes a `foaf:PersonalProfileDocument` with the conventional `foaf:maker` and `foaf:primaryTopic` triples.

**Comment by VirginiaBalseiro** [comment]
Closing in favor of https://github.com/solid/webid-profile/pull/84 (WIP)

**Pull request closed without merging**